CheatSheets/html-cheatsheet.md

```diff
@@ -24,101 +24,4 @@
 | `<style>...</style>`   | The style tag can be used as an alternative to an external style sheet, or complement it.Includes the webpage’s appearance information.   |
 | `<script>...</script>` | Used to add code snippets, typically in JavaScript,to make webpage dynamic. It can also be used to just link to an external script.       |
 
-**[🔼Back to Top](#table-of-contents)**
```

Comment:
You deleted the existing `html` code from [html-cheatsheet.md](https://github.com/crescentpartha/CheatSheets-for-Developers/blob/main/CheatSheets/html-cheatsheet.md)
Please, `restore the previous lines of html code`.